### PR TITLE
Introduce vats/ibc.js handler

### PR DESCRIPTION
**Pull request by michaelfig**
Changes include a new `onBind` event.

extendLoopbackInterfaceHandler to create a handler with implicit
loopback behaviour.

### CODE REVIEW

**Comment by warner** [comment]
@michaelfig I've pushed at attempt at gluing the IBC port to the swingset side. It passes tests but of course cosmic-swingset has barely more than a smoketest at the moment, so you should look at it closely for sanity. I haven't tried any deeper tests yet.

File: packages/cosmic-swingset/lib/chain-main.js [hunk 1]
```diff
@@ -112,26 +120,49 @@ export default async function main(progname, args, { path, env, agcc }) {
...
+      argv,
+    );
     return s;
   }
 
   let blockManager;
   async function toSwingSet(action, _replier) {
     // console.log(`toSwingSet`, action, replier);
     if (action.type === AG_COSMOS_INIT) {
+      // console.error('FIGME: got AG_COSMOS_INIT', action);
```
Comment (kriskowal) [inline packages/cosmic-swingset/lib/chain-main.js:157]: This line is funny.

File: packages/cosmic-swingset/lib/ag-solo/fake-chain.js [hunk 2]
```diff
@@ -42,7 +42,11 @@ export async function connectToFakeChain(basedir, GCI, role, delay, inbound) {
   const vatsdir = path.join(basedir, 'vats');
   const argv = [`--role=${role}`, bootAddress];
   const stateDBdir = path.join(basedir, `fake-chain-${GCI}-state`);
-  const s = await launch(stateDBdir, mailboxStorage, vatsdir, argv);
+  function doOutboundBridge(dstID, obj) {
+    console.error('received', dstID, obj);
```
Comment (kriskowal) [inline packages/cosmic-swingset/lib/ag-solo/fake-chain.js:46]: Is this log left intentionally?

**Review by kriskowal — COMMENTED** [review]
This looks fine progress to my eyes. I don’t see anything alarming, but my review is shallow.

**Review by warner — APPROVED** [review]
also a shallow review, but I didn't see anything to suggest this shouldn't land. we'll need to do a design review with the larger group once it's all landed and working.

**Pull request merged**